test/acceptance/features/bindToService.feature

```diff
@@ -216,3 +216,40 @@ Feature: Bind an application to a service
         Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "True"
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
+
+    Scenario: Bind an imported quarkus app which is deployed as knative service to PostgreSQL database
+        Given Openshift Serverless Operator is running
+        * Quarkus s2i builder image is present
+        * Knative serving is running
+        * DB "db-demo-knative" is running
+        * Quarkus application "knative-app" is imported as Knative service
+        When Service Binding Request is applied to connect the database and the application
+            """
+            ---
```

Comment:
this line could be remove, it separates multiple resources in single yaml file, but here we have only one resource.

Reply:
Done, Thanks